Software/Development/SemesterProject/App/digitalInputManager.c

```diff
@@ -40,9 +42,12 @@ static struct
  */
 void initInputs(void)
```

Comment:
Consider changing name to initDigitalInputs(void)

Reply:
Done